kuma/wiki/events.py

```diff
@@ -60,24 +55,6 @@ def context_dict(revision):
     return context
 
 
-def notification_mails(revision, subject, template, url, users_and_watches):
```

Comment:
Yes. I found no use of it in the code. The last usage was removed a couple years ago by Ricky from what I could find.